apps/shipments/views.py

```diff
@@ -276,6 +284,14 @@ class ListDevicesStatus(APIView):
 
     def get(self, request, *args, **kwargs):
         owner_id = get_owner_id(request)
+
+        LOG.debug(f'Listing devices for owner with id: {owner_id}.')
+        log_metric('transmission.info', tags={'method': 'devices.list', 'module': __name__})
```

Comment:
I'm not  sure we need this additional logging here since we already have a similar on in `get_list_owner_devices`. 
However for consistency, I will suggest to remove the one in `get_list_owner_devices` and keep this one in place. :+1:

Reply:
ah yeah I didn't see it there haha. Yeah, I'll move it for consistency